### ISSUE TITLE
Make sure content can be tabbed through (e.g. About)

### ISSUE THREAD

**Issue by lizzycederberg** [usnationalarchives/nara-bia-photos#252]
The current content is not keyboard accessible. For example, on the about page, a user is unable to tab through the content. Instead, tabbing jumps to the links. We recommend assigning each paragraph or section of text with a tabindex of "0" to include them in the sequential focus during keyboard navigation of the page.
